pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md

```diff
@@ -23,7 +23,33 @@ Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace'
 export WORKSPACE_NAMESPACE=<workspace_namespace>
 ```
 
-## Deploy the application
+After creating a GitRepository, you can either use the Kommander UI or the CLI to deploy your custom applications.
+
+## Deploy the application using the Kommander UI
+
+Go to the Kommander UI to deploy your custom applications:
```

Comment:
```suggestion
Go to the Kommander UI to deploy your catalog applications:
```

Reply:
sounds good. will change this in the new PR.